.github/workflows/inclusive-heat-sensor.yml

```
@@ -0,0 +1,20 @@
+name: Inclusive Heat Sensor
+on:
+  issues:
+    types: [opened, reopened]
+  issue_comment:
+    types: [created, edited]
+  pull_request:
+    types: [opened, reopened]
```

@jonathanpeppers I think this would do that:

```suggestion
  pull_request:
    types: [reopened]
```